### PR TITLE
(#2454) - Ensure .info() call doesnt break taskqueue

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

File: tests/integration/test.replication.js [hunk 1]
```diff
@@ -4122,6 +4122,24 @@ adapters.forEach(function (adapters) {
...
+        var repl = db.replicate.from(remote, {live: true});
+        repl.on('complete', done.bind(null, null));
+
+        remote.info().then(function(results) {
+          repl.cancel();
+        }).catch(function(err) {
+          done(err);
+        });
+      });
+    });
```
Comment (nolanlawson) [inline tests/integration/test.replication.js]: ^ forgot a .catch on the `bulkDocs` promise

Comment (daleharvey) [inline tests/integration/test.replication.js]: Hah good catch, fixed

**Comment by nolanlawson** [comment]
+1

**Comment by daleharvey** [comment]
:) cheers

https://github.com/pouchdb/pouchdb/commit/ed6ee721cde154e168bce420e803fd155eecc676

**Pull request closed without merging**